docs/source/markdown/podman-volume-prune.1.md

```diff
@@ -18,6 +19,17 @@ unused volumes. To bypass the confirmation, use the **--force** flag.
 
 Do not prompt for confirmation.
 
+#### **--filter**
+
+Filter volumes to be pruned. Volumes can be filtered by the following attributes:
+
+- name
+- driver
+- scope
+- label
+- opt
+- dangling
```

Comment:
Alphabetize this list please

Reply:
Fixed: https://github.com/containers/podman/pull/8689/commits/c8a7b9a4ac1aff5e479dd7e333f5aea2bd03dff4